geopandas/tests/test_cmapplotting.py

```diff
@@ -0,0 +1,61 @@
+import sys
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+from shapely.geometry import Point, MultiPoint
+from geopandas import GeoSeries, GeoDataFrame, plotting
+try:
+    if sys.version_info < (2, 7):
+        import unittest2
+    else:
+        raise ImportError()
+except ImportError:
+    import unittest
+
+class TestPlotting(unittest.TestCase):
+    def setUp(self):
+        self.N = 10
+        self.points = GeoSeries(Point(i, i) for i in range(self.N))
+        values = np.arange(self.N)*10
+        self.df = GeoDataFrame({'geometry': self.points, 'values': values})
+        bounds = np.array([-10, 0, 10, 30, 50, 70, 90])
+        self.norm = mpl.colors.BoundaryNorm(boundaries=bounds, ncolors=256)
+
+    def test_boundmin(self):
+        bounds = np.array([-10, 0, 10, 30, 50, 70, 90])
+        norm = mpl.colors.BoundaryNorm(boundaries=bounds, ncolors=256)
+        self.assertEqual(min(self.norm.boundaries), min(norm.boundaries),'Test ran successfully, both min are equal')
```

Comment:
What are you trying to test here? Because `norm` and `self.norm` are created in the same way, so it is expected that the boundaries are the same?

(or is it just to learn writing tests? Which is no problem, but just to know in which detail I comment)

Reply:
Sir, I was trying to write the test where I wanted to compare the norm made in the function with the norm made in def setUp(self).

Reply:
Yes, but both are created in exactly the same way. 

In setup you do:
```
self.norm = mpl.colors.BoundaryNorm(boundaries=bounds, ncolors=256)
```

and in the test:
```
norm = mpl.colors.BoundaryNorm(boundaries=bounds, ncolors=256)
```

So it is expected that both objects are the same.

I think what you want to test is that the `norm` created by geopandas in the `plot` method (accessed from the resulting figure) has certain expected values.